source/AI.cpp

```diff
@@ -1539,6 +1543,20 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 				&& foe->IsDisabled() && (!canPlunder || Has(ship, foe->shared_from_this(), ShipEvent::BOARD)))
 			continue;
 
+		foe->UpdateTargeterStrength();
+		double targeterStrength = foe->GetTargeterStrength();
+		int targeterCount = foe->GetShipsTargetingThis().size();
+
+		// The next two checks only apply if more than two ships are attacking the foe, as well as
+		// if it has not already been selected as a target recently.
+		// Deprioritize this if more than a quarter of your allies' strength is already attacking.
+		if(targeterStrength >= 0.25 * alliedStrength && targeterCount > 2 && !preferredTarget)
+			range += 500;
+
+		// Deprioritize this if more than twice its strength are already attacking it.
+		if(targeterStrength >= 2. * foe->Strength() && targeterCount > 2 && !preferredTarget)
```

Comment:
I think three-to-one odds are more sensible.

```suggestion
		if(targeterStrength >= 3. * foe->Strength() && targeterCount > 3 && !preferredTarget)
```

Reply:
I misread the code before (greater than two is three when its an integer) and have updated this suggestion accordingly.

Reply:
I marked my earlier comment as resolved, but this one still stands.